pkg/shortnames/shortnames.go

```diff
@@ -334,15 +337,14 @@ func Resolve(ctx *types.SystemContext, name string) (*Resolved, error) {
 	// If we have only one candidate, there's no ambiguity. In case of an
 	// empty candidate slices, callers can implement custom logic or raise
 	// an error.
-	if len(resolved.PullCandidates) <= 1 {
+	if len(resolved.PullCandidates) == 1 {
 		return resolved, nil
 	}
 
 	// If we don't have a TTY, act according to the mode.
 	if !terminal.IsTerminal(int(os.Stdout.Fd())) || !terminal.IsTerminal(int(os.Stdin.Fd())) {
 		switch mode {
 		case types.ShortNameModePermissive:
-			// Permissive falls back to using all candidates.
```

Comment:
Was this removal intentional?

Reply:
That was just sloppy me. Thanks for catching!